changelogs/fragments/6180-replace-deprecated-badzipfile.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - "``BadZipfile`` (with a small ``f``) has been deprecated since Python 3.2, use ``BadZipFile`` (big ``F``) instead, added in 3.2 (https://github.com/ansible-collections/community.general/pull/6180)."
```

I think that's way too much detail for a user of the module. How about:
```suggestion
  - "archive - avoid deprecated exception class on Python 3 (https://github.com/ansible-collections/community.general/pull/6180)."
```